src/constants/notifications.ts

```diff
@@ -184,6 +186,12 @@ export type TriggerOrderNotification = {
   type: TriggerOrderNotificationTypes;
 };
 
+export enum OrderSubmissionStatuses {
+  Submitted,
```

Comment:
nit from me, but it feels weird to read "Submitted" with a loading icon by it - seems intuitively like it should be "Submitting"?

<img width="283" alt="Screenshot 2024-04-30 at 6 01 27 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/3318a23b-6789-4fb2-b8bf-57158886d039">

Reply:
agree here

Reply:
will relay to Se, we're modeling after our competition :p

Reply:
keeping submitted, but removing pending (order best effort opened state)

Reply:
sikeeee se was convinced